be/src/runtime/data_stream_recvr.cc

```diff
@@ -102,6 +102,12 @@ class DataStreamRecvr::SenderQueue {
     bool is_finished() const;
 
 private:
+    struct ChunkItem {
+        int64_t size = 0;
+        ChunkUniquePtr chunk_ptr;
+        google::protobuf::Closure* closure = nullptr;
```

Comment:
Would better comment we why put closure and chunk together. and would better comment the closure is what closure.

Reply:
Fixed